### PR TITLE
Add templates to render change annex in module documents

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
As per https://github.com/metanorma/iso-10303-templates/issues/11

### CODE REVIEW

**Comment by manuelfuenmayor** [comment]
Issues to solve:
* `:edition:` document attribute is not accessible within `yaml2text` block via `{edition}`.
* Every module document must have a `changes_paths.yaml`, just like part 1051.

**Comment by ronaldtse** [comment]
The variable “edition” conflicts with the document attribute also called “edition”? But in YAML is it possible to use something like “data.edition” to refer to it?

**Comment by manuelfuenmayor** [comment]
> The variable “edition” conflicts with the document attribute also called “edition”? But in YAML is it possible to use something like “data.edition” to refer to it?

@ronaldtse, no. The issue is: we need access to the  "edition" document attribute *inside* the yaml2text block (not a variable edition).

Outside yaml2text, `:edition:` value can be accessed by doing `{edition}`, but inside, I get empty string.

That is:

```adoc
= Document attributes
:edition: 10


== Clause

{edition} // renders 10

[yaml2text,...,data]
----
{edition} // empty
----

```

**Comment by manuelfuenmayor** [comment]
@ronaldtse this PR is almost ready for merging. What is left is the resolution of this issue to finish the template: https://github.com/metanorma/metanorma-plugin-datastruct/issues/33

File: modules/module_annex_change_history.adoc [hunk 1]
```diff
@@ -7,40 +7,77 @@
...
+{% liquid
+assign schema       = context.schema
+assign path_prefix  = "../../schemas/modules/"
+assign mapping_path = schema | prepend: path_prefix | append: "/mapping.changes.yaml"
+assign arm_path     = schema | prepend: path_prefix | append: "/arm.changes.yaml"
+assign mim_path     = schema | prepend: path_prefix | append: "/mim.changes.yaml"
+%}
+
+// This should be the {edition} but doesn't work inside yaml2text
+{% assign max_version = 10 %}
```
Comment (ronaldtse) [inline modules/module_annex_change_history.adoc]: @manuelfuenmayor can we use the value from `changes.yaml`?

https://github.com/metanorma/iso-10303-srl/blob/32b0046b402672bec34e8bdf41d34d57fe0867ab/documents/iso-10303-1003/changes.yaml#L4

Comment (manuelfuenmayor) [reply modules/module_annex_change_history.adoc]: @ronaldtse `changes.yaml` does not contain the last version of change annex. No worries, I've solved it in https://github.com/metanorma/iso-10303-templates/pull/15/commits/f72618f2fa02bb4f8b3ccfcc3976ff062b28a6af.

**Comment by manuelfuenmayor** [comment]
PR reading for merging.

**Comment by manuelfuenmayor** [comment]
Please, don't merge yet. I still need to make some adjustments. I'll let you know.

File: modules/module_annex_change_history.adoc (hunk 1 above)
Comment (ronaldtse) [reply modules/module_annex_change_history.adoc]: I noticed you've already done this!

**Comment by ronaldtse** [comment]
OK thanks @manuelfuenmayor !

**Comment by manuelfuenmayor** [comment]
Ok @ronaldtse. Ready for merging 👍

**Comment by ronaldtse** [comment]
Thank you @manuelfuenmayor !

**Pull request merged**